filament/src/details/Camera.h

```diff
@@ -205,10 +215,19 @@ struct CameraInfo {
     explicit CameraInfo(FCamera const& camera) noexcept;
     CameraInfo(FCamera const& camera, const math::mat4& worldOriginCamera) noexcept;
 
-    math::mat4f projection;         // projection matrix for drawing (infinite zfar)
+    int eyeCount = 1;
```

Comment:
should be unsigned. Maybe move it to the end of the structure, to try to avoid padding.